.ruby-version

```diff
@@ -1 +1 @@
-ruby 2.2.1
\ No newline at end of file
+ruby-2.2.1
```

Comment:
I got an rvm doesn't know "ruby 2.2.1". Had to add the dash.

Since our shared hosting currently only does 2.1.2 I think we should actually roll it back. If you're on a project with a custom vm you can update it to whatever you want. But I think the template should match our shared environment if that's where most of our projects will be hosted.